Turkey/PlatformId.cs

```diff
@@ -30,7 +34,14 @@ public List<string> GetPlatformIdsFromOsRelease(string[] lines)
                     versionId = versionId.Substring(0, indexOfDot);
                 }
             }
-            var platforms = new string[] { "linux", id, id + versionId };
+            var platforms = new string[] {
+                "linux",
+                "linux" + "-" + architecture,
+                id,
+                id + "-" + architecture,
+                id + versionId,
```

Comment:
`id + versionId + "-" + architecture` is not present here. I can change that if others think this is a good idea.